man/man8/netqtop.8

```diff
@@ -0,0 +1,53 @@
+.TH netqtop 8 "2020-07-30" "USER COMMANDS"
+.SH NAME
+netqtop \- Summarize PPS, BPS, average size of packets and packet counts ordered by packet sizes 
+on each queue of a network interface.
+.SH SYNOPSIS
+.B netqtop [\-n nic] [\-i interval] 
+.SH DESCRIPTION
+netqtop accounts statistics of both transmitted and received packets on each queue of 
+a specified network interface to help developers check if its traffic load is balanced. 
+The result is displayed as a table with columns of PPS, BPS, average size and 
+packet counts in range [0,64), [64, 5120), [512, 2048), [2048, 16K), [16K, 64K). 
+This is printed every given interval (default 1) in seconds.
+
+The tool uses the net:net_dev_start_xmit and net:netif_receive_skb kernel tracepoints. 
+Since it uses tracepoint, the tool only works on Linux 4.7+.
+
+netqtop introduces significant overhead while network traffic is large. See OVERHEAD 
+section below.
+
+.SH REQUIREMENTS
+CONFIG_bpf and bcc
+.SH OPTIONS
+.TP
+\-n NIC
+Specify the network interface card
+.TP
+\-i INTERVAL
+Print results every INTERVAL seconds.
+The default value is 1.
+.SH EXAMPLES
+.TP
+Account statistics of eth0 and output every 2 seconds:
+#
+.B netqtop -n eth0 -i 1
+.SH OVERHEAD
+In performance test, netqtop introduces a overhead of about 30% PPS drop 
+while printing interval is set to 1 second. With different overload of a 
+network interface, the overhead should be steady.
```

Comment:
Thank you so much for reviewing the code and for your advices. From my performance test I noticed that the PPS drop is alway around 30% no matter how much the total PPS is. For example, when I run 32 threads of sockperf on 32 processors and the total PPS is 5608652 msg/sec. It is 3777976 msg/sec when I run 32 sockperf instances and 1 netqtop together. So the PPS drop is 32.64%. If I run sockperf on 1, 2, 4, 8, 16, ..., 64, 104 processors, the total PPS increases but drop is always around 30%.

Reply:
> I don't understand this sentence "With different overload of a network interface, the overhead should be steady." Could you elaborate?

Thank you so much for reviewing the code and for your advices. From my performance test I noticed that the PPS drop is alway around 30% no matter how much the total PPS is. For example, when I run 32 threads of sockperf on 32 processors and the total PPS is 5608652 msg/sec. It is 3777976 msg/sec when I run 32 sockperf instances and 1 netqtop together. So the PPS drop is 32.64%. If I run sockperf on 1, 2, 4, 8, 16, ..., 64, 104 processors, the total PPS increases but drop is always around 30%.

Reply:
I guess you can just say, "This is the case when different number of queues are used. So be mindful of potential packet drop when using this tool."

Reply:
I changed it to "This is the case when different traffic load is applied." because the PPS represents traffic load over NIC not how many queues of the NIC is used. Is this feasible?